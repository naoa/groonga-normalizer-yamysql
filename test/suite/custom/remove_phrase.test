register normalizers/yamysql

table_create remove_phrases TABLE_PAT_KEY ShortText
load --table remove_phrases
[
{"_key": "テスト"}
]

normalize NormalizerYaMySQL "これはテスト" WITH_TYPES|WITH_CHECKS

