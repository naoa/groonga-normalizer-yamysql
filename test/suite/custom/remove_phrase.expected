register normalizers/yamysql
[[0,0.0,0.0],true]
table_create remove_phrases TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
load --table remove_phrases
[
{"_key": "テスト"}
]
[[0,0.0,0.0],1]
normalize NormalizerYaMySQL "これはテスト" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "これは ",
    "types": [
      "hiragana",
      "hiragana",
      "hiragana",
      "null"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      9
    ]
  }
]
